polyglot/piranha/src/config.rs

```diff
@@ -42,6 +42,9 @@ pub(crate) struct CommandLineArguments {
   /// Path to output summary json
   #[clap(short = 'j', long)]
   pub(crate) path_to_output_summary: Option<String>,
+  /// Should Piranha apply the replacements to the source files or not
+  #[clap(short = 'r', long)]
```

Comment:
```suggestion
  #[clap(short = 'd', long)]
```

Reply:
Let's try using a [default value](https://docs.rs/clap/2.26.1/clap/struct.Arg.html#method.default_value ) here and try?  Then we can get rid of the `Option`